### PR TITLE
Use "vscode-json-languageserver" from NPM

### PR DESCRIPTION
**Pull request by jfcherng**
We were forced to build the server from the source because the binary on https://www.npmjs.com/package/vscode-json-languageserver-bin is pretty outdated. But now another published https://www.npmjs.com/package/vscode-json-languageserver (microsoft is one of collaborators) works in my test. This change also allows the depbot to take care of dependency update.

### CODE REVIEW

File: language-server/package.json [hunk 1]
```diff
@@ -1,37 +1,5 @@
...
-  "author": "Microsoft Corporation",
-  "license": "MIT",
-  "engines": {
-    "node": "*"
-  },
-  "bin": {
-    "vscode-json-languageserver": "./bin/vscode-json-languageserver"
-  },
-  "main": "./out/node/jsonServerMain",
   "dependencies": {
```
Comment (rchl) [inline language-server/package.json:2]: Wouldn't be too bad to keep some of the fields that are "required" (or trigger warning when missing).

**Review by rchl — CHANGES_REQUESTED** [review]
For some reason it appears to behave differently - the trailing commas are reported as warnings in ST preferences, for example. Looks like schemas are not applied.

**Comment by jfcherng** [comment]
```diff
--- a/plugin.py
+++ b/plugin.py

    @@ -211,8 +211,8 @@ class LspJSONPlugin(NpmClientHandler, StoreListener):
             self._schema_store.load_schemas()
     
         @request_handler('vscode/content')
    -    def handle_vscode_content(self, uri: str, respond: Callable[[Any], None]) -> None:
    -        respond(self._schema_store.get_schema_for_uri(uri))
    +    def handle_vscode_content(self, uris: List[str], respond: Callable[[Any], None]) -> None:
    +        respond(next((self._schema_store.get_schema_for_uri(uri) for uri in uris), None))
     
         # ST4-only
         def on_pre_send_notification_async(self, notification: Notification) -> None:
    @@ -224,7 +224,7 @@ class LspJSONPlugin(NpmClientHandler, StoreListener):
         # --- StoreListener ------------------------------------------------------------------------------------------------
     
         def on_store_changed(self, schemas: List[Dict]) -> None:
    -        self._api.send_notification('json/schemaAssociations', schemas + self._user_schemas)
    +        self._api.send_notification('json/schemaAssociations', [schemas + self._user_schemas])
     
     
     class LspJsonAutoCompleteCommand(sublime_plugin.TextCommand):
```

The interface of `Schema content request`/`Schema associations notification` didn't seem to be changed. Don't know why I have to do this to make it work...

**Comment by predragnikolic** [comment]
> The original https://github.com/vscode-langservers/vscode-json-languageserver/tree/master/src is now force pushed to be a CSS server somehow 

So strange.

**Comment by rchl** [comment]
Can you address it this way?:

```diff
diff --git a/plugin.py b/plugin.py
index 97dfa1b..de20ac9 100644
--- a/plugin.py
+++ b/plugin.py
@@ -1,7 +1,7 @@
 from abc import ABCMeta, abstractmethod
 from LSP.plugin import DottedDict
 from LSP.plugin import Notification
-from LSP.plugin.core.typing import Any, Callable, Dict, List, Optional
+from LSP.plugin.core.typing import Any, Callable, Dict, List, Optional, Tuple
 from lsp_utils import ApiWrapperInterface
 from lsp_utils import request_handler
 from lsp_utils import NpmClientHandler
@@ -211,8 +211,8 @@ class LspJSONPlugin(NpmClientHandler, StoreListener):
         self._schema_store.load_schemas()
 
     @request_handler('vscode/content')
-    def handle_vscode_content(self, uri: str, respond: Callable[[Any], None]) -> None:
-        respond(self._schema_store.get_schema_for_uri(uri))
+    def handle_vscode_content(self, params: Tuple[str], respond: Callable[[Any], None]) -> None:
+        respond(self._schema_store.get_schema_for_uri(params[0]))
 
     # ST4-only
     def on_pre_send_notification_async(self, notification: Notification) -> None:
@@ -224,7 +224,7 @@ class LspJSONPlugin(NpmClientHandler, StoreListener):
     # --- StoreListener ------------------------------------------------------------------------------------------------
 
     def on_store_changed(self, schemas: List[Dict]) -> None:
-        self._api.send_notification('json/schemaAssociations', schemas + self._user_schemas)
+        self._api.send_notification('json/schemaAssociations', [schemas + self._user_schemas])
 
 
 class LspJsonAutoCompleteCommand(sublime_plugin.TextCommand):
```

I can't push to your repo as usual :)

**Review by rchl — APPROVED** [review]

**Pull request merged**